packages/widget/src/hooks/use-make-wallets.tsx

```diff
@@ -215,6 +226,50 @@ export const useMakeWallets = () => {
           isWalletConnected: connector.id === currentConnector?.id,
         };
 
+        if (chainType === 'cosmos' && chainID === 'pacific-1') {
+          minimalWallet.getAddress = async ({ signRequired, context }) => {
+            const address = await evmGetAddress({ signRequired, context });
+            if (!address) {
+              toast.error(
+                <p>
+                  <strong>Failed to get address!</strong>
+                </p>
+              );
+            }
+            const publicClient = createPublicClient({
+              chain: sei,
+              transport: http(),
+            });
+            try {
+              const seiAddress = await publicClient.readContract({
+                args: [address as `0x${string}`],
+                address: '0x0000000000000000000000000000000000001004',
+                abi: seiPrecompileAddrABI,
+                functionName: 'getSeiAddr',
+              });
+              toast.success(
+                `Successfully retrieved ${context} address from ${connector.name}`
+              );
+              return seiAddress;
+            } catch (error) {
+              console.error(error);
+              toast.error(
+                <p>
+                  Your EVM address (0x) has not associated on chain yet. Please
+                  visit{' '}
+                  <a
+                    target="_blank"
+                    className="underline"
+                    href="https://app.sei.io/"
+                  >
+                    https://app.sei.io/
+                  </a>{' '}
+                  to associate your SEI address.
+                </p>
+              );
```

Comment:
@codingki can we make sure this stays open until they click on the link? also add a button to dismiss it manually

Reply:
yeah, will add

Reply:
![image](https://github.com/user-attachments/assets/1dde7fa6-9884-4c00-b22f-bdd28814e4a6)